docs/walkthrough.md

```diff
@@ -50,132 +51,92 @@ When we create this `ClusterServiceBroker` resource, the service catalog control
 by querying the broker server to see what services it offers and creates a
 `ClusterServiceClass` for each.
 
-We can check the status of the broker using `kubectl get`:
+We can check the status of the broker using `svcat describe`:
 
 ```console
-$ kubectl get clusterservicebrokers ups-broker -o yaml
```

Comment:
I would like to keep this content in - we should show both kubectl and svcat, and explain where svcat gets you more than just kubectl.

Reply:
@pmorie  Do you have a suggestion for how to show/mention the two separate commands? People didn't like the "run one of the commands" lead ins for all the code blocks.

Maybe just say at the top that we are following a specific convention and that they can run either command provided? That way we don't have to awkwardly explain the dual commands each time?

Reply:
I feel like we should have a separate doc for `kubectl` workflow, and recommend that folks follow the `svcat` flow

Reply:
If our docs were on a real site that was javascript enabled (like on a githubpages site, instead of github renderings of markdown source) we could display a single code block with tabs for the preferred tool, e.g. 

```
| *svcat* | kubectl |
$ svcat command
svcat output
```